lib/puppet-strings/yard/handlers/ruby/function_handler.rb

```diff
@@ -17,6 +17,7 @@ class PuppetStrings::Yard::Handlers::Ruby::FunctionHandler < PuppetStrings::Yard
     required_block_param
     optional_block_param
     return_type
+    example
```

Comment:
`example` isn't a parameter method, is it? you can remove this.

Reply:
You're right, it was a debug leftover.